### PR TITLE
BAPI using SNC Connection

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/partners/sap/sap-bapi-connector.md [hunk 1]
```diff
@@ -169,15 +175,29 @@ This example shows setting of below properties to GetJCoDestination action:
 
 {{< figure src="/attachments/partners/sap/sap-bapi-connector/getjcodestination-additional.properties.png" >}}
 
-The microflow actions are described in the following sections.
+#### 6.1.1 Using an SNC Connection
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-bapi-connector.md]: This should go in parallel to section 4.1#SAP System Connection Details.
so connecting to SAP system using SNC is an alternative way to provide connection details for setting development environment using Studio Pro.
We still have to find out how to run it on SAP BTP, so for now let's mention it for running in Studio pro for local set up

File: content/en/docs/partners/sap/sap-bapi-connector.md [hunk 2]
```diff
@@ -169,15 +175,29 @@ This example shows setting of below properties to GetJCoDestination action:
...
+
+Alternatively, you can configure **GetJCoDestination** to use an SNC connection by setting up the properties listed below (using the method described in [Additional JCoProperties](#jco-properties)), and then passing the list of properties as described above.
+
+The values for the properties can be found in the **Connection Properties** of the BAPIs which you can see by right-clicking the BAPI in the **SAP GUI for Java**. The options from the **Expert Settings** can be found in the **Advanced** tab as a string which will resemble a list of HTTP parameters: `conn=/R/DA1/G/PUBLIC&sncname=p:CN=ehdev.industrysoftware.automation.siemens.com,C=US,0=Siemens,OU=Siemens&sncon=true&sncqop=9&cpg=1100&dcpg=4110&clnt=340&lang=EN&systemName=DA1`.
```
Comment (gauravkrshah) [inline content/en/docs/partners/sap/sap-bapi-connector.md]: These are not connection properties of BAPI, it is connection properties of SAP server. so we can put it like:
values for the properties can be found in the **Connection Properties** of the SAP server to want to connect. Go to SAP GUI (don't mention of Java) and view "connection Properties" of the SAP server to connect. 
blurred the parameter values as this is specific to Siemens system, not to make public.

**Review by gauravkrshah — CHANGES_REQUESTED** [review]
Hi Mark, kindly look at review comments and let me know if any more details required.

**Comment by MarkvanMents** [comment]
@gauravkrshah 
Hi Gaurav,
I've made some changes based on your comments. Does it look better now? Thanks.

**Review by gauravkrshah — APPROVED** [review]
Looks Good

**Comment by MarkvanMents** [comment]
Update already released - publication approved over Slack.

**Pull request merged**